source/js/components/nsf-nav.js

```diff
@@ -32,9 +32,14 @@ var MenuLinks = React.createClass({
   },
   onScroll: function() {
     var links = this.state.links || [];
+    var hash = window.location.hash.replace(`#`, ``);
     var active = links[0] || ``;
     var scrollY = window.scrollY;
 
+    if (hash && !document.querySelector(`.nav-anchor#` + hash)) {
```

Comment:
In this case I'm ensuring the current URL hash is a nav anchor, and not a accordion anchor.

If the user scrolls while an accordion is open, we don't want the url to change.

I didn't think it would be worth it to try to "figure out" if we should close the accordion on scroll or not.